projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -33,9 +33,19 @@ public function register_routes() {
 			$this->namespace,
 			$this->rest_base,
 			array(
-				'methods'             => WP_REST_Server::READABLE,
-				'callback'            => array( $this, 'get_data' ),
-				'permission_callback' => array( $this, 'can_read' ),
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_data' ),
+					'permission_callback' => array( $this, 'can_access' ),
+				),
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => array( $this, 'update_site_options' ),
+					'permission_callback' => array( $this, 'can_access' ),
+					'request_format'      => array(
```

Comment:
I could not find any mention of `request_format` in Core's v2 architecture. Is that a carry-over from v1.1?